src/Function/Bundles.agda

```diff
@@ -20,16 +20,16 @@
 module Function.Bundles where
 
 open import Function.Base using (_∘_)
+open import Function.Consequences.Propositional
+  using (strictlySurjective⇒surjective; strictlyInverseˡ⇒inverseˡ; strictlyInverseʳ⇒inverseʳ)
```

Comment:
It is tempting to want to inline these, as in the `Propositional` case, they take on a particularly simple form which is obscured by the three levels of indirection in these definitions...

Reply:
inline these where?

Reply:
Firstly, in `Function.Consequences.Propositional` (see latest commit), but one could even imagine inlining these definitions (given how simple they are) at any call-site...?

Reply:
Last words on this:
* I think it is better, given that the types of the corresponding lemmas in `Function.Consequences.Propositional` *differ* from those in `Function.Consequences{.Setoid}` (by making `f` explicit), we can, moreover should, feel free to reimplement them, as is now done; 
* the explicit quantification seems necessary in the handful of call-sites where these lemmas are actually used (and with enough of them, that I no longer propose inlining them there)...
* ... BUT is wrong for `strictlyInverseʳ⇒inverseʳ` where it is `f⁻¹` which should be supplied explicitly, and not `f`! Fixing this should form part of #2570